tools/check-git-version.js

```diff
@@ -0,0 +1,24 @@
+const shell = require('shelljs');
+
+function checkGitVersion() {
+  try {
+    const gitVersion = shell
+      .exec('git --version', { silent: true })
+      .stdout.toString()
+      .slice(12)
+      .split('.');
+    if (
+      parseInt(gitVersion[0], 10) >= 2 &&
+      parseInt(gitVersion[1], 10) >= 22 &&
+      parseInt(gitVersion[2], 10) >= 0
+    ) {
+      process.exit(0);
+    } else {
+      throw new Error('Minimum git version 2.33.0 is required');
```

Comment:
```suggestion
      throw new Error(`Minimum git version ${minimumGitVersion} is required`);
```
Better to define the minimum version at the top of the file and then reuse it in strings/comparisons later